include/hexahedron_element.h

```diff
@@ -167,7 +192,21 @@ class HexahedronElement : public Element<Tdim> {
   //! \retval indices Indices that form sub-tetrahedrons
   Eigen::MatrixXi inhedron_indices() const override;
 
+  //! Return indices of a face of an element
+  //! \param[in] face_id given id of the face
+  //! \retval indices Indices that make the face
+  Eigen::VectorXi face_indices(unsigned face_id) const override;
+
  private:
+  //! Namespace containing constants of face id
+  const std::map<unsigned, Eigen::VectorXi> face_indices_hexahedron{
```

Comment:
This is a private variable so end with `_`

Reply:
Great! Nice design making this a const private

Reply:
Thanks. Understood. I forgot about `_`